interfaces/HO-Portal/src/global.scss

```diff
@@ -107,3 +116,39 @@ ion-button {
 ion-icon {
   pointer-events: none;
 }
+
+#portal-title {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  height: 48px;
+
+  #logo-container {
+    margin: 0 4px 0 0;
+    padding: 0;
+    width: 48px;
+  }
+
+  h1 {
+    margin-top: 10px;
+    margin-left: 4px;
+    letter-spacing: 3px;
+  }
+}
```

Comment:
We could make separate component out of this even maybe... (but maybe not now per se.. ;) )